platform/binance/api.go

```diff
@@ -16,6 +17,7 @@ type Platform struct {
 
 func (p *Platform) Init() error {
 	p.client.BaseURL = viper.GetString("binance.api")
+	p.client.BaseDexURL = viper.GetString("binance.dex")
```

Comment:
Add corresponding configuration to `config.yml` as far as Binance DEX RPC is public https://dex.binance.org/api/v1